.github/workflows/nightly.yaml

```diff
@@ -91,8 +91,22 @@ jobs:
           DATE="$(date +%Y-%m-%d)"
           echo "tag=nightly-${DATE}" >> $GITHUB_OUTPUT
 
+      - name: Build Unittest Container
+        uses: docker/build-push-action@v4
+        with:
+          load: true
+          build-args: |
+            CMAKE_BUILD_TYPE=Release
+            S3GW_VERSION=${{ steps.date.outputs.tag }}
+            NPROC=16
+          tags: |
```

Comment:
Nit: line break unnecessary

Reply:
The docker/build-push-action parses these things in a special way:
https://github.com/docker/build-push-action#inputs
This works and I'm not keen on fumbling around to make it work without the line break.